configure.ac

```diff
@@ -543,6 +543,16 @@ AS_IF([test x"$with_uring" != x"no"],
 			[])],
       [])
 
+AS_IF([test "$have_liburing" = "1"], [
+	# Requires liburing >= 2.1 for multishot support
+	save_CPPFLAGS="$CPPFLAGS"
+	AS_IF([test $with_uring != ""]
+	      [CPPFLAGS="$CPPFLAGS -I$with_uring/include/"])
+	AC_CHECK_DECLS([io_uring_prep_poll_multishot, IORING_CQE_F_MORE], [],
+		       [have_liburing=0], [[#include <liburing.h>]])
+	CPPFLAGS="$save_CPPFLAGS"
+])
```

Comment:
What about the AC_DEFINE_UNQUOTED([HAVE_LIBURING... above? Shouldn't this be done only when have_liburing == 1?